patches/tModLoader/Terraria.IO/WorldFile.cs.patch

```diff
@@ -150,6 +150,23 @@
  				{
  					writer.Write(nPC2.active);
  					writer.Write(nPC2.netID);
+@@ -2181,14 +_,14 @@
+ 				int num3 = reader.ReadInt32();
+ 				Tile tile = Main.tile[num2, num3];
+ 				Sign sign;
+-				if (tile.active() && Main.tileSign[(int)tile.type])
++				if (true) //if (tile.active() && Main.tileSign[(int)tile.type]) 
+ 				{
+ 					sign = new Sign();
+ 					sign.text = text;
+ 					sign.x = num2;
+ 					sign.y = num3;
+ 				}
+-				else
++				else 
```

Comment:
Nice space :')

Reply:
Reeeee I was so careful to check lmao!!!!!!